packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -88,29 +44,43 @@ class DefaultController extends AdminBaseController
      * @param \Shopsys\FrameworkBundle\Component\Grid\GridFactory $gridFactory
      * @param \Shopsys\FrameworkBundle\Component\Cron\Config\CronConfig $cronConfig
      * @param \Shopsys\FrameworkBundle\Component\Cron\CronFacade $cronFacade
+     * @param \Shopsys\FrameworkBundle\Model\AdminNavigation\BreadcrumbOverrider $breadcrumbOverrider
+     * @param \Shopsys\FrameworkBundle\Twig\DateTimeFormatterExtension|null $dateTimeFormatterExtension
      */
     public function __construct(
-        StatisticsFacade $statisticsFacade,
-        StatisticsProcessingFacade $statisticsProcessingFacade,
-        MailTemplateFacade $mailTemplateFacade,
-        UnitFacade $unitFacade,
-        Setting $setting,
-        AvailabilityFacade $availabilityFacade,
-        CronModuleFacade $cronModuleFacade,
-        GridFactory $gridFactory,
-        CronConfig $cronConfig,
-        CronFacade $cronFacade
+        protected readonly StatisticsFacade $statisticsFacade,
+        protected readonly StatisticsProcessingFacade $statisticsProcessingFacade,
+        protected readonly MailTemplateFacade $mailTemplateFacade,
+        protected readonly UnitFacade $unitFacade,
+        protected readonly Setting $setting,
+        protected readonly AvailabilityFacade $availabilityFacade,
+        protected readonly CronModuleFacade $cronModuleFacade,
+        protected readonly GridFactory $gridFactory,
+        protected readonly CronConfig $cronConfig,
+        protected readonly CronFacade $cronFacade,
+        protected ?BreadcrumbOverrider $breadcrumbOverrider,
```

Comment:
In that case these two arguments should maybe also have default `null`.

Reply:
Changed